docs/preview/03-Features/powershell/bicep.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Bicep Templates"
+layout: default
+---
+
+# Bicep
```

Comment:
Maybe an idea to create a separate folder for templates, like we did with Azure storage, so that the ARM and Bicep stuff are together instead of spread:
![image](https://user-images.githubusercontent.com/9039753/211310121-d3d6a9de-05d8-4145-8612-b678df932a2f.png)

Reply:
Changed in [6a44c09 (#357)](https://github.com/arcus-azure/arcus.scripting/pull/357/commits/6a44c095a8a0439593029593e6ad67ed924d398a)